README.md

```diff
@@ -217,14 +223,6 @@ Questions can be asked by raising an `Issue`.
 - [Udokaku Ugochukwu](https://github.com/UdokaVrede)
 - [Usman Tatla](https://github.com/iTatla1)
 - [Vamsi Ravi](https://github.com/vamsi963601)
+- [Victory Chiamaka Wekwa](https://github.com/VictoryWekwa)
 - [Vinicius Ramos](https://github.com/redrookie)
 - [Vivek Singh](https://github.com/vivmost)
-- [Victory Chiamaka Wekwa](https://github.com/VictoryWekwa)
-- [Stephen Mount (@stemount)](https://github.com/stemount)
-- [Asharib Ahmed](https://github.com/Asharib90)
-- [Michael Akintomo (@MadeByM1KE)](https://github.com/Makintomo04)
-- [Anubhav Purohit](https://github.com/Anubhav-byte)
-- [Mitanshi Kshatriya](https://github.com/MitanshiKshatriya)
-- [Ganapathi Subramanyam Jayam](https://github.com/ganapathi12)
-- [Aditya Palpattuwar](https://github.com/APalpattuwar7)
-- [Kehinde Adeleke](https://github.com/adeleke5140)
```

Comment:
I think some did not get added back in?

7 added and 9 removed?

Reply:
I removed two names cause they were already on the list. Probably they added there names twice.

Reply:
Oh good spot, thank you 👍

Reply:
Here are the two names :
Asharib Ahmed
Michael Akintomo (@MadeByM1KE)